scripts/test/system.sh

```diff
@@ -110,6 +110,25 @@ EOF
     echo "::endgroup::"
 }
 
+function test_subctl_recover_broker_info() {
+    # Rename old broker-info.subm to be able to compare it with the recovered broker-info.sum
+    echo "Renaming broker-info.subm to broker-info.subm.orig"
+    mv "${DAPPER_SOURCE}"/output/broker-info.subm "${DAPPER_SOURCE}"/output/broker-info.subm.orig
+    base64 -d "$DAPPER_SOURCE"/output/broker-info.subm.orig > "$DAPPER_SOURCE"/output/decoded_broker_info.subm.orig
+
+    _subctl recover-broker-info --context cluster1
+    cmp_rm_files
+
+    _subctl recover-broker-info --context cluster2
+    cmp_rm_files
+}
+
+function cmp_rm_files() {
```

Comment:
If we keep this, it should have a name indicating that it’s specific to `broker-info.subm`, and indicates its purpose rather than its implementation, _e.g._ `validate_and_clean_broker_info`.